app/src/main/java/com/mobile/tandil/kotlinbaseproject/utils/Constants.kt

```diff
@@ -2,5 +2,8 @@ package com.mobile.tandil.kotlinbaseproject.utils
 
 object Constants {
     const val LISTENER_PARKING_SPACES_SETTING_KEY: String = "LISTENER_PARKING_SPACES_SETTING_KEY"
+    const val LISTENER_SET_DATE: String = "LISTENER_SET_DATE"
+    const val LISTENER_SET_TIME: String = "LISTENER_SET_TIME"
+    const val IS_START_KEY: String = "IS_START_KEY"
     const val EMPTY_STRING_KEY: String = ""
```

Comment:
please change this name from EMPTY_STRING_KEY to EMPTY_STRING

Reply:
done